services/order.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict],
+    username: str,
+    date: datetime = None,
+) -> None:
+
+    order = Order.objects.create(
+        user=get_user_model().objects.get(username=username)
+    )
+
+    for ticket in tickets:
+        if date:
+            order.created_at = date
+            order.save()

```

Comment:
Setting the `created_at` field manually might not be necessary, as it is automatically set by Django when the `Order` instance is created. Consider removing this logic unless there is a specific requirement to set a custom creation date.